lib/index.js

```diff
@@ -31,15 +31,15 @@ internals.implementation = function (server, options) {
       if (!token && request.auth.mode === 'optional') {
         return reply.continue({ credentials: {} });
       }
-      
+
       if (!token) {
-        return reply(Boom.unauthorized('Missing auth token'));
+        return reply(Boom.unauthorized(null, 'Token'));
```

Comment:
Ok. Not seen two auth strategies required for the same request/route but I'm all ears. Can you describe the scenario in a bit more detail so we understand the use-case? we'd *love* to help you solve this.